### PR TITLE
Default search order

### PR DESCRIPTION
**Pull request by ptbrowne**
Fixes https://github.com/visualize-admin/visualization-tool/issues/811

- feat: Default sort order is newest, when a search is made, it switches to relevance

### CODE REVIEW

File: app/graphql/resolvers/rdf.ts [hunk 1]
```diff
@@ -33,7 +32,37 @@ import {
...
+    case undefined:
+    case null:
+      results.sort((a, b) => {
+        const ra = getCube(a).datePublished || "0";
+        const rb = getCube(b).datePublished || "0";
+        return descending(ra, rb);
+      });
+      break;
+    case DataCubeResultOrder.Score:
+      break;
```
Comment (bprusinowski) [inline app/graphql/resolvers/rdf.ts:59]: Results are sorted be score by default, right?

**Review by bprusinowski — COMMENTED** [review]
Overall LGTM, but the behaviour is inconsistent when removing the search query and clicking enter and when clicking "x" to clear the search query – in the first case it sorts by Newest, while in second it sorts by Title. I feel like they should behave in the same way, what do you think?

**Comment by bprusinowski** [comment]
Also, not related to this PR, but do you know how it's possible that some cubes do not have a created date?

File: app/graphql/resolvers/rdf.ts (hunk 1 above)
Comment (ptbrowne) [reply app/graphql/resolvers/rdf.ts:59]: Yes, when they enter this function, they are already sorted.

**Comment by ptbrowne** [comment]
No, I don't know :/ @l00mi ?

**Comment by ptbrowne** [comment]
>  I feel like they should behave in the same way, what do you think?

Agreed, I improved the behavior, and fixed the test ;)

**Review by bprusinowski — APPROVED** [review]
Thanks, works great now 🎉

**Comment by l00mi** [comment]
> No, I don't know :/ @l00mi ?

It should not happen, but if cubes are manually created, it might always happen, that this gets forgot, or is "not yet" available. So it is up to you to either ignore these cubes, or handle the fact that they have no created date.

**Comment by ptbrowne** [comment]
OK we handle this case. Do you have documentation on which SHACL rules are applied when ? It seems like the ICV guard mode from Stardog could be useful to apply validation, is this something that could be activated to improve data integrity ?

**Comment by l00mi** [comment]
Lets discuss this in one of the next dev meetings.

**Pull request merged**